### PR TITLE
add compat data for SVGGeometryElement

### PR DESCRIPTION
**Pull request by solfen**
Link: https://developer.mozilla.org/en-US/docs/Web/API/SVGGeometryElement

### CODE REVIEW

File: api/SVGGeometryElement.json [hunk 1]
```diff
@@ -0,0 +1,344 @@
...
+              "version_added": null
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": "53"
```
Comment (maboa) [inline api/SVGGeometryElement.json]: Table here shows it as not being supported https://developer.mozilla.org/en-US/docs/Web/API/SVGGeometryElement however following the bug (https://bugzilla.mozilla.org/show_bug.cgi?id=1325320) in the note we find that it became supported in 61, I assume for both desktop and mobile versions. @Elchi3 should be able to confirm.

File: api/SVGGeometryElement.json [hunk 2]
```diff
@@ -0,0 +1,344 @@
...
+      },
+      "supportOtherThanPath": {
+        "__compat": {
+          "description": "Support for elements other than <code>&lt;path&gt;</code>",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": null
```
Comment (maboa) [inline api/SVGGeometryElement.json]: Maybe @jpmedley can give us the status of the chrome family?

File: api/SVGGeometryElement.json [hunk 3]
```diff
@@ -0,0 +1,344 @@
...
+              "version_added": true
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": "53"
```
Comment (maboa) [inline api/SVGGeometryElement.json]: Note, this comes with a note. Judging by the bug in the note (https://bugzilla.mozilla.org/show_bug.cgi?id=1325320) looks like the full support was added in version 61.

You might want to represent something like:
```
"firefox": [
  {
    "version_added": "53",
    "notes": "This property is currently only supported on <code><path></code> elements (where it was already supported in earlier Firefox versions through the <code>SVGPathElement</code> interface). Support for other elements will be added in <a href='...'>bug 1325320</a>."
  },
  {
    "version_added": "61"
  }
]
```

File: api/SVGGeometryElement.json [hunk 4]
```diff
@@ -0,0 +1,344 @@
...
+            },
+            "firefox_android": {
+              "version_added": "53"
```
Comment (maboa) [inline api/SVGGeometryElement.json]: Similarly for `firefox_android` - again @Elchi3 should be able to confirm.

File: api/SVGGeometryElement.json [hunk 5]
```diff
@@ -0,0 +1,344 @@
...
+              "version_added": true
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": "53"
```
Comment (maboa) [inline api/SVGGeometryElement.json]: Again from the note this should be fixed for version 61 - please add note to 53 and reflect support in 61 - something like:

```
"firefox": [
  {
    "version_added": "53",
    "notes": "This property is currently only supported on <code><path></code> elements (where it was already supported in earlier Firefox versions through the <code>SVGPathElement</code> interface). Support for other elements will be added in <a href='...'>bug 1325320</a>."
  },
  {
    "version_added": "61"
  }
]
```

File: api/SVGGeometryElement.json [hunk 6]
```diff
@@ -0,0 +1,344 @@
...
+            },
+            "firefox_android": {
+              "version_added": "53"
```
Comment (maboa) [inline api/SVGGeometryElement.json]: Please also add the note above and indicate support for Firefox 61. Thanks :)

File: api/SVGGeometryElement.json [hunk 7]
```diff
@@ -0,0 +1,344 @@
...
+              "version_added": true
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": "53"
```
Comment (maboa) [inline api/SVGGeometryElement.json]: Again, from the note this should be fixed for version 61 - please add note to 53 and reflect support in 61 - something like:

```
"firefox": [
  {
    "version_added": "53",
    "notes": "This property is currently only supported on <code><path></code> elements (where it was already supported in earlier Firefox versions through the <code>SVGPathElement</code> interface). Support for other elements will be added in <a href='...'>bug 1325320</a>."
  },
  {
    "version_added": "61"
  }
]
```

File: api/SVGGeometryElement.json [hunk 8]
```diff
@@ -0,0 +1,344 @@
...
+            },
+            "firefox_android": {
+              "version_added": "53"
```
Comment (maboa) [inline api/SVGGeometryElement.json]: Please also add the note above and indicate support for Firefox 61. Thanks :)

File: api/SVGGeometryElement.json (hunk 1 above)
Comment (Elchi3) [reply api/SVGGeometryElement.json]: Yeah, 61 is correct here. I updated that.

File: api/SVGGeometryElement.json (hunk 5 above)
Comment (Elchi3) [reply api/SVGGeometryElement.json]: I think it is fine to have a sub feature at the interface level about this and then not have these notes appear everywhere for the methods/properties.

**Pull request merged**